### ISSUE TITLE
[bitnami/discourse] Allow to specify DISCOURSE_DB_BACKUP_PORT

### ISSUE THREAD

**Issue by lbyte00** [bitnami/charts#4456]
**Which chart**:
bitnami/discourse latest

**Describe the bug**
At the moment it's not possible to specify DISCOURSE_DB_BACKUP_PORT which is used by Discourse for backup purposes.

```
[2020-11-22 11:18:24] pg_dump: [archiver (db)] connection to database "******" failed: could not connect to server: Connection timed out
[2020-11-22 11:18:24] Is the server running on host "******" (143.110.161.47) and accepting
[2020-11-22 11:18:24] TCP/IP connections on port 5432?
[2020-11-22 11:18:24] Unpausing sidekiq...
[2020-11-22 11:18:24] EXCEPTION: pg_dump failed
```

Container image doesn't support the var:
https://github.com/discourse/discourse/blob/f8305f53c787ad0b6cd865d56a7b8b0f77f9dcc2/config/discourse_defaults.conf#L40-L41

**To Reproduce**
Setup postgresql instance having not 5432 port and try to perform Discourse backup using this DB.

**Expected behavior**
Use of specified in values.yaml DB port during backup in Discourse.
Successful backup.

**Comment by miguelaeh** [comment]
Hi @lbyte00 ,
Thank you very much for requesting this feature. Would you like to create a PR to it? We will be pleased to review it.

**Comment by lbyte00** [comment]
Hi @miguelaeh!

**Comment by lbyte00** [comment]
Hi @miguelaeh!

I could create the PR for this chart to add the dedicated variable, though as I see it's required to also change the Docker image to support[ that var](https://github.com/discourse/discourse/blob/f8305f53c787ad0b6cd865d56a7b8b0f77f9dcc2/config/discourse_defaults.conf#L40-L41), because I tried to pass the var through extraEnvVars and the container doesn't see it.

I'm not sure where and how should the container be changed, if you could link some example then yes, I can create the PR for that too :)

**Comment by miguelaeh** [comment]
Hi @lbyte00 ,
Thank you very much for offering to create the PRs.
I checked and this container seems to be based on the nami tool. It will be difficult for you to make the changes, so you can create the PR for the Chart and link this issue, and we will modify the container on our side.
Thanks!

**Comment by lbyte00** [comment]
Hi @miguelaeh,

Thank you for handling the container part.
Finally got time to create the PR for this chart, looking forward to getting a review!

**Comment by miguelaeh** [comment]
Thank you very much, we will review it soon. Apart from that, I am going to create an internal task to handle the container part. Nevertheless, I am afraid I cannot prove an estimated date for this feature. We will update the PR and this thread once we have news.
Thank you very much!!
Regards

**Comment by marcosbc** [comment]
Hi, FYI we just released a chart version, [4.0.0](https://github.com/bitnami/charts/tree/master/bitnami/discourse#to-400). The initialization logic has received a major refactor, and scripts for the container image are now located in the [`rootfs/` directory](https://github.com/bitnami/bitnami-docker-discourse/blob/master/2/debian-10/rootfs/).

I'm unsure if this is an environment variable, in which case you can use [`extraEnvVars`/`extraEnvVarsCM`/`extraEnvVarsSecret`](https://github.com/bitnami/charts/blob/master/bitnami/discourse/values.yaml#L148), i.e.:

```
discourse:
  extraEnvVars:
    - name: DISCOURSE_DB_BACKUP_PORT
      value: ...
```

If it is an option to be added to `discourse.conf`, you can enable this new configuration via the following environment variable instead:

- `DISCOURSE_EXTRA_CONF_CONTENT`: Extra configuration to append to the `discourse.conf` configuration file. No defaults.
